webapp/src/ts/modules/tasks/tasks-content.component.ts

```diff
@@ -144,7 +144,7 @@ export class TasksContentComponent implements OnInit, OnDestroy, AfterViewInit {
     this.globalActions.settingSelected();
 
     return this
-      .hydrateTaskEmission(task)
+      .hydrateTaskEmission(task, id)
```

Comment:
Can't you take the id from the task? Why the extra param?

Reply:
Good catch! Now getting it from the task.